### PR TITLE
Release/7.0.1

### PR DESCRIPTION
**Pull request by PeterSiegl**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 7.0.1
+
+* Small `Lookup` enhancements
```
Comment (fspoettel) [inline CHANGELOG.md]: Please be a bit more specific here ^^

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-lds",
-  "version": "7.0.0",
+  "version": "7.0.1",
```
Comment (fspoettel) [inline package.json]: This is not a bugfix so should get `7.1.0` version imo

File: CHANGELOG.md (hunk 1 above)
Comment (PeterSiegl) [reply CHANGELOG.md]: kk

File: package.json (hunk 2 above)
Comment (PeterSiegl) [reply package.json]: just some minor stuff but kk

Comment (fspoettel) [reply package.json]: https://semver.org/#spec-item-7 https://semver.org/#spec-item-6

**Pull request closed without merging**